plugins/task/checkfiles/checkfiles.php

```diff
@@ -130,8 +132,13 @@ protected function checkImages(ExecuteTaskEvent $event): int
 				$this->logTask('PLG_TASK_CHECK_FILES_LOG_IMAGE_SAVE_FAIL', 'error');
 			}
 
-			// We do at most a single resize per execution
-			break;
+			--$numImages;
+
+			// We do a limited number of resize per execution
+			if ($numImages <= 0)
```

Comment:
```suggestion
			if ($numImages == 0)
```
Will it ever be less than 0?

Reply:
No, you're right.